packages/fxa-graphql-api/src/auth/gql-customs.guard.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
+import { GqlExecutionContext } from '@nestjs/graphql';
+import { Request } from 'express';
+import { CustomsService } from 'fxa-shared/nestjs/customs/customs.service';
+
+import { SessionTokenResult } from './session-token.strategy';
+
+@Injectable()
+export class GqlCustomsGuard implements CanActivate {
+  constructor(private customs: CustomsService) {
+    if (!customs) {
+      throw new Error('No customs service provided.');
```

Comment:
Huh, I thought Nest's dependency injection took care of this for us?

Reply:
same here, but due to how guards work it seems that it did not locate it when loading in one case, such that I had to add this